MODULE.bazel

```diff
@@ -16,3 +16,5 @@ bazel_dep(name = "maliput", version = "1.1.1")
 bazel_dep(name = "tinyxml2", version = "9.0.0")
 
 bazel_dep(name = "googletest", version = "1.14.0", dev_dependency = True)
+
+local_path_override(module_name="maliput", path="../maliput")
```

Comment:
Worth for local use case but probably we don't want to add this `local_path_override` statement to the repo, right?

Reply:
Aye, fixed.